src/DataViews/include/DataViews/ModulesDataView.h

```diff
@@ -21,7 +23,8 @@ namespace orbit_data_views {
 class ModulesDataView : public DataView {
  public:
   explicit ModulesDataView(AppInterface* app,
-                           orbit_metrics_uploader::MetricsUploader* metrics_uploader);
+                           orbit_metrics_uploader::MetricsUploader* metrics_uploader,
+                           bool new_ui = absl::GetFlag(FLAGS_devmode));
```

Comment:
Nit: Not sure if AppInterface has this, but there is App::IsDevMode.

Reply:
AppInterface doesn't have `IsDevMode`, but ModulesDataView is instantiated from OrbitApp where it is available. I removed the default parameter and used `IsDevMode`